<ui version="4.0" >
 <class>IMapQMAPExport</class>
 <widget class="QDialog" name="IMapQMAPExport" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export map ...</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Output path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="labelPath" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolPath" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>File prefix</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="linePrefix" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="lineDescription" />
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="pushCancel" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="pushExport" >
       <property name="text" >
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushCancel</sender>
   <signal>clicked()</signal>
   <receiver>IMapQMAPExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>800</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel" >
     <x>423</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
